README.md

```diff
@@ -1,2 +1,26 @@
 # AWSClusterManagersDemo.jl
-Demo for AWSClusterManagers.jl presented at JuliaCon2021
+## Overview
+This demo project was presented at JuliaCon2021 to showcase a minimal example of the [AWSClusterManagers.jl](https://github.com/JuliaCloud/AWSClusterManagers.jl) package on AWS Batch. The job submitted to Batch calculates the approximate value of PI. The explanation for this can be found in this [article](https://jccraig.medium.com/calculate-pi-with-a-dartboard-bdb433f1c999).
+
+The job will:
+1. Launch four worker nodes
+2. Have each worker run a small process
+3. Retrieve and combine the results
+4. Display the results to CloudWatch logs
+## Setup
+**Note: When running this package you will be responsible for all AWS charges in your account.**
+
+To launch this yourself:
+
+1. Store your AWS account ID in an variable
+   	`AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)`
+1. Create a new ECR (Elastic Cloud Registry) repository to host you Docker image as:
+   `aws cloudformation foobar`
```

Comment:
Should this be something like `aws cloudformation deploy --template-file ecr_template.yml`?

Reply:
Yes, whoops! Lol